config.example.yaml

```diff
@@ -60,6 +60,16 @@ services:
                     retry_delay: 250
                     retry_limit: 10
                     show_sql: false
+                  citoid:
+                    host: http://citoid-beta.wmflabs.org
+                  mathoid:
+                    host: http://mathoid-tester.wmflabs.org
+                  mobileapps:
+                    host: http://appservice.wmflabs.org
+                  # 10 days Varnish caching, one day client-side
+                  purged_cache_control: s-maxage=864000, max-age=86400
+                  # Cache control for purged endpoints allowing short-term client caching
+                  purged_cache_control_client_cache: s-maxage=1209600, max-age=300
```

Comment:
Unless we know that no external (and un-purged) proxies see the s-maxage header, it might be better to set the value to 0. This way, we can be sure that there is no surprising caching in transparent proxies when the wiki is accessed via HTTP.